pkg/crc/machine/start.go

```diff
@@ -290,10 +290,14 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 	}
 
 	// Start network time synchronization if `CRC_DEBUG_ENABLE_STOP_NTP` is not set
-	if stopNtp, _ := strconv.ParseBool(os.Getenv("CRC_DEBUG_ENABLE_STOP_NTP")); !stopNtp {
-		logging.Info("Starting network time synchronization in CodeReady Containers VM")
-		if _, _, err := sshRunner.Run("sudo timedatectl set-ntp on"); err != nil {
-			return nil, errors.Wrap(err, "Failed to start network time synchronization")
+	if stopNtp, _ := strconv.ParseBool(os.Getenv("CRC_DEBUG_ENABLE_STOP_NTP")); stopNtp {
+		logging.Info("Stopping network time synchronization in CodeReady Containers VM")
+		if _, _, err := sshRunner.RunPrivileged("timedatectl set-ntp off"); err != nil {
+			return nil, errors.Wrap(err, "Failed to stop network time synchronization")
+		}
+		logging.Info("Setting clock to 3 months ahead")
+		if _, _, err := sshRunner.RunPrivileged("date -s '3 month'"); err != nil {
+			return nil, errors.Wrap(err, "Failed to set clock to 3 months ahead")
```

Comment:
I'd prefer that we sync the current host time instead of hardcoding '3 months', this would make this code slightly more flexible.